CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+# v0.17.4
+
+## What's Changed
```

Comment:
This is grabbed from the release note auto generation.
https://github.blog/2021-10-04-beta-github-releases-improving-release-experience/